app/src/test/java/waiter/EchoServerTest.java

```diff
@@ -52,7 +52,7 @@ void catchesSocketException() {
     void callsCommunicate() {
         CommunicatorMock communicatorMock = new CommunicatorMock(false, new IOException());
         EchoServer echoServer = new EchoServer(communicatorMock);
-        echoServer.start(4425);
+        echoServer.start(4424);
```

Comment:
It changed before because there was a issue with using the same port in sequential tests, but it was the result of finding that I wasn't cleaning up the serverSocket properly. So I changed it back to 4424 when I put the ServerSocket new up in a try with resources. I hope this made sense! 😄